clang/lib/Driver/ToolChains/Flang.cpp

```diff
@@ -151,6 +151,8 @@ void Flang::addCodegenOptions(const ArgList &Args,
       !stackArrays->getOption().matches(options::OPT_fno_stack_arrays))
     CmdArgs.push_back("-fstack-arrays");
 
+  Args.AddLastArg(CmdArgs, options::OPT_fexperimental_loop_fuse, options::OPT_fno_experimental_loop_fuse);
```

Comment:
Not nit: Probably need to run `clang-format` on this line.

NIT
```suggestion
  Args.AddLastArg(CmdArgs, options::OPT_fexperimental_loop_fusion, options::OPT_fno_experimental_loop_fusion);
```